src/main/scala/gitbucket/core/api/CreateARepositoryWebhook.scala

```diff
@@ -0,0 +1,35 @@
+package gitbucket.core.api
+
+case class CreateARepositoryWebhookConfig(
+  url: String,
+  content_type: String = "form",
+  insecure_ssl: String = "0",
+  secret: Option[String]
+)
+
+/**
+ * https://docs.github.com/en/rest/reference/repos#create-a-repository-webhook
+ */
+case class CreateARepositoryWebhook(
+  name: String = "web",
+  config: CreateARepositoryWebhookConfig,
+  events: List[String] = List("push"),
```

Comment:
Is this hard coded to push?  

In the GUI, the following are supported:

Create - Branch, or tag created.
Gollum - Wiki page updated.
Issue comment - Issue commented on.
Issues - Issue opened, closed.
Pull request - Pull request opened, closed, or synchronized.
Pull request review comment - Pull request diff commented on.
Push - Git push to a repository.

Reply:
This is the default value of the case class. You can specify other values when you call API to create a webhook.

Reply:
Ok, thanks, I tried that and it didn't work so I opened ticket:  https://github.com/gitbucket/gitbucket/issues/2596

Reply:
@solvingj please see https://github.com/gitbucket/gitbucket/issues/2596#issuecomment-726113271